components/studio/bin/hab-studio.sh

```diff
@@ -1069,12 +1052,12 @@ cleanup_studio() {
 
 # **Internal**  Run a command which may fail without aborting whole script
 try() {
-  debug "TRYing '$*'"
+  >&2 echo "TRYing '$*'"
```

Comment:
You could drop this statement I think--I would show up every single time the function is called and be reported to the user whether or not the command was successful.